src/pages/HomeSelector.vue

```diff
@@ -130,6 +130,7 @@ watch(isLoading, async (newValue) => {
 const { idle } = useIdle(20 * 60 * 1000); // 10 min
 const confirm = useConfirm();
 const timeLeft = ref(60);
+const { t } = useI18n();
```

Comment:
This seems like the only component where you explicitly import `useI18n` and declare the `t` variable. Is this necessary since you used `app.use(i18n)` in the main.js file?

Reply:
Similar to the other issue, the string lliteral syntax {{ $t('someTranslation.item') }} is only defined within the HTML template. In the script, we have to create a specific instance of i81n-vue and perform the translation that way.

I am of course open to a better solution.